### PR TITLE
chore: Deployment flow

### PR DESCRIPTION
**Pull request by Yserz**

### CODE REVIEW

File: aws/application.py [hunk 1]
```diff
@@ -48,9 +48,19 @@ def index():
 
 @application.route('/auth/')
 @main.latest_browser_required
+def auth_new():
```
Comment (lipis) [inline aws/application.py]: leave that `auth`

File: aws/application.py [hunk 2]
```diff
@@ -48,9 +48,19 @@ def index():
...
+  response = flask.make_response(flask.render_template(
+    'auth/index_new.html',
+    country=util.geoip_country(),
+  ))
+  return response
+
+
+@application.route('/auth/old/')
+@main.latest_browser_required
 def auth():
```
Comment (lipis) [inline aws/application.py]: make it `auth_old`

File: grunt/tasks/misc.js [hunk 3]
```diff
@@ -96,11 +97,11 @@ module.exports = grunt => {
     'copy:deploy_favicon',
     'includereplace:prod_index',
     'includereplace:prod_auth',
+    'includereplace:prod_auth_new',
     'clean:deploy_app',
     'uglify:deploy',
     'concat:deploy',
     'clean:deploy_script',
-    'raygun:prod',
```
Comment (lipis) [inline grunt/tasks/misc.js:103]: If we actually don't need it anymore.. let's remove the whole task to clean up things

Comment (Yserz) [reply grunt/tasks/misc.js:103]: Done

**Pull request merged**